ordinary_data/functions/fn_element_part_set_orientation.sql

```diff
@@ -0,0 +1,108 @@
+CREATE OR REPLACE FUNCTION qwat_od.fn_element_part_set_orientation(_element_id integer)
```

Comment:
this looks similar to https://github.com/qwat/qwat-data-model/blob/master/ordinary_data/functions/fn_valve_set_orientation.sql
would there be a chance to avoid code duplication?

Reply:
Yes it is quite similar, but as it is not the same object there are some differences. Maybe we could avoid code duplication but it needs more thinking about it (maybe consider other objects that need to be oriented) ?